data/brands/shop/mall.json

```diff
@@ -81,6 +81,8 @@
       "locationSet": {"include": ["au", "nz"]},
       "tags": {
         "brand": "Westfield",
+        "brand:wikidata": "Q18378979",
+        "brand:wikipedia": "en:Westfield Group",
```

Comment:
This Wikipedia article and Wikidata item are about a former company. The two sites are littered with the remnants of former iterations of the company. To confuse things further, I created [this item](https://www.wikidata.org/wiki/Q106623253) to represent the brand per se, which can be added to both this NSI entry and the Euro-American one above.

Reply:
Thanks.  I wasn't sure if that was an appropriate solution or not from the Wikidata perspective. I'll update the PR to use that instead.